### PR TITLE
Footer language dropdown changes

### PR DESCRIPTION
**Pull request by cantsin**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/318)

## What does this change?

- Moves language selection to "Contact" section
- Styles dropdown
- Adds icons
- Mobile friendly

## Screenshots (for front-end PR):

(Still subject to revision)

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by cantsin** [comment]
One known issue: IE11 language button is oddly styled. still not quite sure why.

File: crt_portal/static/sass/custom/footer.scss [hunk 1]
```diff
@@ -65,3 +91,26 @@ footer {
...
+    background-color: $gold;
+    color: black;
+    margin-top: 0;
+  }
+
+  .i8n-submit {
+    margin-left: 1em;
+    margin-top: 0 !important;
+    background-color: #2e2e2a;
+    border: 1px solid white;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:109]:
```suggestion
    width: 100%; // IE11 fix
```

File: crt_portal/static/sass/custom/footer.scss [hunk 2]
```diff
@@ -9,33 +9,60 @@ footer {
   }
 }
 
+.usa-footer__contact {
+  display: flex;
+  flex-direction: column;
+
+  @include at-media-max(desktop) {
+    margin-bottom: 40px;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:17]:
```suggestion
    margin-bottom: 2.5rem;
```

File: crt_portal/static/sass/custom/footer.scss [hunk 3]
```diff
@@ -9,33 +9,60 @@ footer {
...
+  }
+}
+
+.usa-footer__section {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 24px;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:24]:
```suggestion
  margin-bottom: 1.5rem;
```

File: crt_portal/static/sass/custom/footer.scss [hunk 4]
```diff
@@ -9,33 +9,60 @@ footer {
...
+
+  &:last-child {
+    margin-top: 8px;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:27]:
```suggestion
    margin-top: 0.5rem;
```

File: crt_portal/static/sass/custom/footer.scss [hunk 5]
```diff
@@ -9,33 +9,60 @@ footer {
...
+    align-items: center;
+  }
+}
+
 .usa-footer__icon {
-  padding-right: 0 !important;
+  margin-right: 24px;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:33]:
```suggestion
  margin-right: 1.5rem;
```

File: crt_portal/static/sass/custom/footer.scss [hunk 6]
```diff
@@ -9,33 +9,60 @@ footer {
...
 }
 
 .usa-footer__logo-section {
-  margin-top: 1em;
-  margin-bottom: 2em;
+  margin-top: 1.25em;
+  margin-bottom: 2.5em;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:38]:
```suggestion
  margin-top: 1.25rem;
  margin-bottom: 2.5erm;
```

File: crt_portal/static/sass/custom/footer.scss [hunk 7]
```diff
@@ -9,33 +9,60 @@ footer {
...
 }
 
 .usa-footer__logo-heading {
   font-size: 2em;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:42]:
```suggestion
  font-size: 2rem;
```

File: crt_portal/static/sass/custom/footer.scss [hunk 8]
```diff
@@ -9,33 +9,60 @@ footer {
...
   font-weight: bold;
+
+  @include at-media-max(desktop) {
+    font-size: 22px;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:46]:
```suggestion
    font-size: 1.38rem;
```

File: crt_portal/static/sass/custom/footer.scss (hunk 1 above)
Comment (cantsin) [reply crt_portal/static/sass/custom/footer.scss:109]: D'oh. Yes.

File: crt_portal/static/sass/custom/footer.scss [hunk 9]
```diff
@@ -9,33 +9,60 @@ footer {
...
 .usa-footer__heading {
   font-size: 1.5em;
   font-weight: bold;
+  margin-top: 0;
   margin-bottom: 1em;
 }
 
 .usa-footer__separator {
-  margin-top: 1em;
+  margin-top: 16px;
```
Comment (hbillings) [inline crt_portal/static/sass/custom/footer.scss:65]:
```suggestion
  margin-top: 1rem;
```

**Comment by LindsayYoung** [comment]
the text is getting cut off for me on chrome and safari, I did hard refreshs and rebuilt assets. 
Screen Shot 2020-05-13 at 4 35 47 PM

**Pull request merged**